### PR TITLE
fix hub clusterClaim is not created at the beginning

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: agent/pkg/controllers/hubclusterclaim_controller.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+func (c *hubClusterClaimController) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.Result, error) {
+	reqLogger := c.log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	reqLogger.V(2).Info("hub clusterClaim controller", request.NamespacedName)
+
+	var err error
+	mch, err := getMCH(ctx, c.client, request.NamespacedName)
+	if err != nil {
+		reqLogger.Error(err, "failed to get MCH instance")
```
Comment (clyang82) [inline agent/pkg/controllers/hubclusterclaim_controller.go:31]: You may ignore the mch is not found error.

Comment (zhiweiyin318) [reply agent/pkg/controllers/hubclusterclaim_controller.go:31]: check it in getMCH.

Comment (clyang82) [reply agent/pkg/controllers/hubclusterclaim_controller.go:31]: Got it. Thanks

**Pull request merged**